code/modules/admin/verbs/debug.dm

```diff
@@ -861,6 +861,20 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
 /proc/cmp_timer_data(list/a, list/b)
 	return b["count"] - a["count"]
 
+/client/proc/spawn_abnormality_boss()
+	set category = "Admin.Fun"
+	set name = "Spawn Abnormality Event Boss"
+	//The order and contents of the bosslist should match the defines set in lobotomy_events
+	var/list/bosslist = list("Apocalypse Bird","Ying and Yang","The Jester of Nihil") //TBA : General bee, nobody is.
+	if(!SSticker.HasRoundStarted())
+		alert("Wait until the game starts")
+		return
+	var/M = input(usr,"Who would you like to spawn? (This will take a few seconds to load)","Select a boss") as null|anything in bosslist
+	if(!M)
+		return
+	SSlobotomy_events.SpawnEventAbnos(bosslist.Find(M))
+	SSblackbox.record_feedback("tally", "admin_verb", 1, "Spawned abnormality boss [M]")
+
```

Comment:
Appreciated; Though I believe one of two things should be done here. Either the `bosslist` becomes associative with the defines or equivalent defines of the event defines in the Lobotomy Corp Events subsystem OR a comment is here noting that the index of the list is equivalent to the defined value in the Lobotomy Corp Events subsystem.

Reply:
There is already a comment, "//The order and contents of the bosslist should match the defines set in lobotomy_events". However, I agree that this is normally a valid concern.